resources/compass/charts/cockpit/values.yaml

```diff
@@ -3,7 +3,8 @@ images:
     path: eu.gcr.io/kyma-project
   ui:
     dir:
-    version: ab79f919
+    version: 56a4db12
```

Comment:
`ab79f919` is a newer commit than `56a4db12`, why did you change it back?

Reply:
thanks for spotting that, this was a version used in https://github.com/kyma-incubator/compass/blob/master/chart/compass/charts/cockpit/values.yaml
I revert that change.